src/core/blob.cpp

```diff
@@ -14,13 +14,12 @@
 using namespace soci;
 
 blob::blob(session & s)
+	: backEnd_(std::unique_ptr<details::blob_backend>(s.make_blob_backend()))
```

Comment:
Why not just
```suggestion
	: backEnd_{s.make_blob_backend()}
```
?

Reply:
Because it doesn't really matter and I'm not used to using brace initialization :shrug:

Reply:
Well, using braces was just part of it (and is a good practice as it avoids narrowing conversions, so I don't agree that it doesn't really matter), but this was mostly about omitting constructing a temporary object unnecessarily.

Reply:
> constructing a temporary object unnecessarily

Ah yeah - that will be optimized away anyway, but yeah it's a bit verbose. I'll change it.